src/tests/Microsoft.PowerFx.Interpreter.Tests/InternalSetupTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.PowerFx.Core.Parser;
+using Microsoft.PowerFx.Core.Tests;
+using Microsoft.PowerFx.Interpreter.Tests.Helpers;
+using Xunit;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class InternalSetupTests : PowerFxTest
+    {
+        [Fact]
+        public void InternalSetup_Parse_Null()
+        {
+            var iSetup = InternalSetup.Parse(null);
+
+            Assert.NotNull(iSetup);
+            Assert.Null(iSetup.HandlerName);
+            Assert.Equal(TexlParser.Flags.None, iSetup.Flags);
+        }
+
+        [Fact]
+        public void InternalSetup_Parse_EmptyString()
+        {
+            var iSetup = InternalSetup.Parse(string.Empty);
+
+            Assert.NotNull(iSetup);
+            Assert.Null(iSetup.HandlerName);
+            Assert.Equal(TexlParser.Flags.None, iSetup.Flags);
+        }
+
+        [Fact]
+        public void InternalSetup_Parse_Whitespaces()
+        {
+            var iSetup = InternalSetup.Parse("  ");
+
+            Assert.NotNull(iSetup);
+            Assert.Null(iSetup.HandlerName);
+            Assert.Equal(TexlParser.Flags.None, iSetup.Flags);
+        }
+
+        [Fact]
+        public void InternalSetup_Parse_SimpleHandler()
+        {
+            var iSetup = InternalSetup.Parse("SomeHandler");
+
+            Assert.NotNull(iSetup);
+            Assert.Equal("SomeHandler", iSetup.HandlerName);
+            Assert.Equal(TexlParser.Flags.None, iSetup.Flags);
+        }
+
+        [Fact]
+        public void InternalSetup_Parse_OneFlag()
+        {
+            var iSetup = InternalSetup.Parse(TexlParser.Flags.NamedFormulas.ToString());
```

Comment:
>TexlParser.Flags.NamedFormulas.ToString() [](http://example.com/codeflow?start=45&length=41)

from a test perspective - at least one of these cases should be a string literal. 

This change now makes the C# identifier name part of our "public" test contract. Ie, if somebody renamed the C# enum value, it could now break tests.  
So adding as a literal would explicitly catch that.

Reply:
(as an example, see the LSP tests like TestLogCallbackExceptions - they pass in literals for "method" even though we have symbols).

Reply:
Added a test with a string